src/pyhf/tensor/numpy_backend.py

```diff
@@ -38,8 +38,12 @@ class numpy_backend(object):
 
     def __init__(self, **kwargs):
         self.name = 'numpy'
-        self.float_precision = '64b'
-        self.int_precision = '64b'
+        self.precision = kwargs.get('precision', '64b')
+        self.dtypemap = {
+            'float': np.float64 if self.precision == '64b' else np.float32,
+            'int': np.int64 if self.precision == '64b' else np.int32,
+            'bool': np.bool_,
```

Comment:
@kratsg one question: Why is this `np.bool_` (the underscore)?

Reply:
This is just a "Matthew doesn't know" question, not a "this is a problem" question.

Reply:
https://stackoverflow.com/a/6205237/1532974 -- this is a numpy thing.

Reply:
Answer: https://stackoverflow.com/questions/6205020/numpy-types-with-underscore-int-float-etc/6205237#6205237